src/benchmark.cc

```diff
@@ -424,6 +428,12 @@ bool BenchmarkFamilies::FindBenchmarks(
         if (family->arg_count_ >= 2) {
           AppendHumanReadable(instance.arg2, &instance.name);
         }
+        if (!IsZero(family->min_time_)) {
+          instance.name +=  StringPrintF("/min_time:%0.3f",  family->min_time_);
```

Comment:
The way we calculate the width seems weird. Shouldn't it just be the size of the longest name plus room for `_stddev`?